playbook-website/app/javascript/components/Website/src/pages/README.md

```diff
@@ -0,0 +1,5 @@
+# Pages Folder
```

Comment:
This is great. But knowing Show is going to get complex...do you want to provide a folder here showing how to group by page?

Reply:
I had that in the beginning but put back because the scope shifted to be much smaller based on our conversation. 

If you have a mockup that is finished, I can definitely take a look, and spec out the structure a little more here. 

Is the latest somewhere here? https://www.figma.com/file/8GHYnjKWp7jbUmCWWa7Yvt/Playbook-Site-Vision-2023?type=design&node-id=711-19827&mode=design&t=U25L77kO2e67v0ln-0